src/app/features/addresses/components/atoms/AddressDisplay/AddressDisplay.tsx

```diff
@@ -23,12 +23,17 @@ const AddressDisplay: React.FC<Props> = ({ bagId, headingSize = "h2", isHeader =
   const { data } = useBAG(bagId)
   const { data: otherAddresses } = useOtherAddressesByBagId(bagId)
 
+  const isCurrentAddress = (address: any) => address.adres.trim() === data?.results[0].adres.trim()
```

Comment:
Does `const isCurrentAddress = (address: { adres: string }) => address.adres.trim() === data?.results[0].adres.trim()` work?

Reply:
Yes, it does :-)
Any doubts?

Reply:
Not if it works :)